docs/AboutUs.md

```diff
@@ -31,14 +31,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Trinh Hoai Song Thu (Sarah)
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/sarah.png" width="200px">
```

Comment:
I think the name of the file needs to match your github username. 
See: "The filename of the profile photo should be docs/images/github_username_in_lower_case.png" from the 2103 website

Reply:
oh thank you!